src/components/Perils/PerilItem/PerilItem.tsx

```diff
@@ -0,0 +1,161 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import axios from 'axios'
+import {
+  LAPTOP_BP_UP,
+  MOBILE_BP_UP,
+  TABLET_BP_UP,
+} from 'components/blockHelpers'
+import { PerilIcon } from 'components/Perils/types'
+import React, { useEffect, useState } from 'react'
+
+interface PerilItemProps {
+  title: React.ReactNode
+  description: string
+  icon: PerilIcon
+}
+
+const OuterContainer = styled.div`
+  color: ${colorsV3.gray900};
+`
+
+const Container = styled.button`
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: flex-start;
+  text-align: left;
+  min-height: 3.75rem;
+  padding: 0.5rem 1rem 0.5rem 0.375rem;
+  color: inherit;
+  font-family: inherit;
+  border-radius: 0.375rem;
+  border: 0;
+  background-color: ${colorsV3.gray100};
+  cursor: pointer;
+  transition: all 150ms ease-in-out;
+  -webkit-appearance: none;
+
+  :focus {
+    outline: none;
+  }
+
+  &:focus {
+    box-shadow: none;
+    outline: none;
+  }
+
+  ${TABLET_BP_UP} {
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0.5rem;
+
+    :hover {
```

Comment:
same, shouldn't this really be `&:hover`? Otherwise if i'm not mistaken this might target childrens' hovers

Reply:
Yep hasty mistakes 🤦‍♂️